modules/component/src/component.module.ts

```diff
@@ -1,15 +1,15 @@
 import { NgModule } from '@angular/core';
-import { LetDirective } from './let/let.directive';
-import { Async$Pipe } from './push$/async$.pipe';
-import { Push$Pipe } from './push$/push$.pipe';
+import { LetDirective } from './let';
+import { PushPipe } from './push';
 
-const DECLARATIONS = [LetDirective, Push$Pipe, Async$Pipe];
+export const DECLARATIONS = [LetDirective, PushPipe];
 
-const EXPORTS = [DECLARATIONS];
+export const EXPORTS = [DECLARATIONS];
 
 @NgModule({
   declarations: [DECLARATIONS],
   imports: [],
+  // @TODO ivy compatible
   exports: [EXPORTS],
 })
 export class NgRxComponentModule {}
```

Comment:
```suggestion
export class ComponentModule {}
```

We don't prefix with `NgRx`

Reply:
done